### PR TITLE
Improve existing instructions of creating a developer instance

### PR DESCRIPTION
**Pull request by ultimatecoder**
* **Description:** According to my recent experience I found instructions provided for creating a developer instance is weak. It is almost impossible to create running instance of this project without taking any help from a Hasgeek Hacker. I am putting some efforts in managing clean documentation a bit. Please let me know what contributors are thinking about this note.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -6,25 +6,55 @@ Code for HasGeek's Funnel at http://funnel.hasgeek.com/
...
 
-  $ python manage.py db create
+2. Copy `instance/settings-sample.py` to `instance/development.py` using below
+   command::
 
+        $ cp instance/settings-sample.py instance/development.py
 
-WSGI is recommended for production. Enable ``mod_wsgi`` in Apache and make a
-``VirtualHost`` with::
+3. You have to register your application with Haskgeek for required
```
Comment (jace) [inline README.rst]: `HasGeek`

File: README.rst [hunk 2]
```diff
@@ -6,25 +6,55 @@ Code for HasGeek's Funnel at http://funnel.hasgeek.com/
...
+   `instance/development.py` file with below credentials obtained from
+   https://auth.hasgeek.com.
+   
+   For example abtained **Client access key** is 
+   `IpdA1NGuSri5E2ceVbCkcw` and **Client secret** is
+   `uIWeb_2NQgeMeuU2HaRnZQBPstoRQUSnKyL5GpSEMpvw` then updated values in
+   your `instance/development.py` should look like below::
+
+        LASTUSER_CLIENT_ID = 'IpdA1NGuSri5E2ceVbCkcw'
+        LASTUSER_CLIENT_SECRET = 'uIWeb_2NQgeMeuU2HaRnZQBPstoRQUSnKyL5GpSEMpvw'
```
Comment (jace) [inline README.rst]: Are these real credentials? I hope they aren't.

File: README.rst [hunk 3]
```diff
@@ -6,25 +6,55 @@ Code for HasGeek's Funnel at http://funnel.hasgeek.com/
...
+
+5. You may also want to setup the database with::
+
+        $ python manage.py db upgrade head
```
Comment (jace) [inline README.rst]: Use `python manage.py createdb`. Never run `db upgrade` on a new installation.

File: README.rst (hunk 2 above)
Comment (ultimatecoder) [reply README.rst]: I have "obfuscated" the real one.

**Comment by ultimatecoder** [comment]
@jace Updated the instruction according to your past comment.

**Comment by jace** [comment]
Some of the steps have changed. @iambibhas is currently working on Funnel and should review and update the instructions.

**Pull request closed without merging**